content/docs/start/experiments/experiment-pipelines.md

```diff
@@ -1,14 +1,15 @@
 ---
-title: 'Get Started: Building Pipelines'
+title: 'Get Started: Experiment Pipelines'
```

Comment:
I think you're hitting a much larger issue here (design? product?). one that get-started doc cannot fully resolve, and def not in this PR :) why do we have `dvc exp run` and not `dvc pipeline run` ? 😉 
Using the experiment commands and terminology as synonyms to pipelines is indeed super confusing... this is why @daavoo and @dacbd are in favour of consolidating the pipeline contents to its own section. which would make sense for this aspect, but would create another discrepancy where the pipeline content won't be strongly associated with experiment section...
It's a deeper issue indeed.

> Making exps reproducible

This is not a good title for get started I think. it's too "advanced", focused on a feature of a thing and not the thing. The deeper issue here is that we don't have several names for pipelines: data-pipelines, pipelines and experiments

Reply:
To trace back my mental steps, I initially called this: **experiments as pipelines**
But it didn't look as clear to me as a title / section and page name, so I condensed it a bit 😄

Reply:
so, let's do just "pipelines" for now? My point is - let's not invent and reinvent wheels please (like live tracking, etc). It's important to keep it simple. Any adjective immediately raises a lot of questions. It's better to be very explicit and "dumb" vs smart and eloquent, etc. I hope that makes sense.

Again, in this case I would personally go with some actual task that we are solving. Even pipelines are too abstract to my mind for this section.

Reply:
> so, let's do just "pipelines" for now?

The point of this PR is to get some order and logic into those chapters so they make sense in the context of the trails and be meaningful. This chapter deals with using pipelines for experiments and using `dvc exp run` on them. It has to be distinct from the data pipelines chapter under data-management and associated with experiments somehow to be meaningful and clear.
This is not inventing or re-inventing anything here, just removing vagueness and confusion that hit you when you're reading this page today.

I will go with "Experimenting using pipelines" in this case, and will have to rename other chapters to make them a bit more consistent, it will make them less compact.
I don't think it's better but that's the only other way to reconcile our differences here I think. I truely think that naming this "pipelines" with the current structure is counter productive. 

>  It's important to keep it simple. Any adjective immediately raises a lot of questions.

I am a firm believer that clarity are more important than brevity 😄 

> Again, in this case I would personally go with some actual task that we are solving. Even pipelines are too abstract to my mind for this section.

I'm sorry I'm not getting this part I guess. This is not a real life tutorial but get-started, but I think "experimenting using pipelines" is maybe close to what you meant here. If I understand correctly

Reply:
> I am a firm believer that clarity is more important than brevity 😄

yes 100%, imo, and in this case Building Pipelines is more clear vs "Experiments Pipelines". That's more or less the essence of my comment. 

> This is not inventing or re-inventing anything here, just removing vagueness and confusion that hit you when you're reading this page today.

if you are trying to optimize a case when person lands to this page and you want them to see "Experiments" (it's already in URL, it should hopefully clear from the text that this belongs to a trail, but anyways), I would make the _title_ (not the sidebar entry), something like "Experiments: Building Pipelines" (just an example).

My point in all this discussion (similar to the `live ..`) is that any adjective or modifier double the mental effort needed to understand this. E.g. "Simple Pipelines" (are there complex? what's the difference, etc). In this case I read "Experiments" as modifier (not a as context) and it raises questions in my head. 

> It has to be distinct from the data pipelines chapter under data-management and associated with experiments somehow to be meaningful and clear.

It doesn't have to I think, btw. They both could have the same title - and I would be personally fine with that. I think we are bit overcomplicating this. Also, clearly pipelines is a topic on which we don't have a clear decision / discussion yet.

Reply:
I understand, we may not being in agreement about the role of modifiers and adjectives though.
To try and move this forward I pushed the change so it is now "Experimenting using pipelines".
If/when we condense the content further so "experiments" will be a single page, or multi-page but with a specific entry points, and then "pipelines" or "building pipelines" will just be a part of it in a clear context, I would agree with this change, but currently, with chapters being self-sufficient I want the differentiation to be very clear.

> clearly pipelines is a topic on which we don't have a clear decision / discussion yet.

Many discussions from what I've heard ;) not a decision

PTAL at the changes now, tell me if we're good to go

Reply:
it's ready when you feel it's ready :)

> but currently, with chapters being self-sufficient I want the differentiation to be very clear

yep, as a reminder - there are ways to this by making the title (what you on see on the page, not sidebar) very explicit - you have way more space there and it's good for SEO.